espnet2/enh/espnet_enh_s2t_model.py

```diff
@@ -187,9 +191,12 @@ def forward(
         speech_ref = None
         if self.calc_enh_loss:
             assert "speech_ref1" in kwargs
-            speech_ref = [
-                kwargs["speech_ref{}".format(spk + 1)] for spk in range(num_spk)
-            ]
+            speech_ref = []
+            for spk in range(num_spk):
+                speech_ref.append(kwargs["speech_ref{}".format(spk + 1)])
+                if text.dim() == 3:
+                    # Set batch-wise dummy speaker reference to very small value
+                    speech_ref[spk][torch.where(spk >= batch_num_spk)] = 0.0005
```

Comment:
Why is 0.0005 is chosen here? I'm curious what is the original value for `speech_ref[spk]` when spk >= batch_num_spk in data preparation.
Another suggestion is to define some `eps` value at the top of this file and use it elsewhere.